### PR TITLE
Make bootstrap default

### PR DESCRIPTION
**Pull request by DavidKang**
For 2.10 release, we plan to set the Bootstrap interface by default. For that, we had to adapt some part of the code base:
 - Put bootstrap as default in the feature switch
 - Replace support file `feature_bootstrap` by `feature_bento`
 - Adapt tests
 - Adapt Circle-CI configuration

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -192,7 +192,10 @@ jobs:
       - run: *bootstrap_old_test_suite
       - run:
           name: Run spider
-          command: cd src/api; bundle exec rake test:spider
+          command: |
+            cd src/api
+            export BENTO=1
+            bundle exec rake test:spider
```
Comment (hennevogel) [inline .circleci/config.yml]: Hm, why do we run the spider test with bento?

Comment (DavidKang) [reply .circleci/config.yml]: Because we never run the spider test with Bootstrap. Shall I adapt the spider test for it?

Comment (hennevogel) [reply .circleci/config.yml]: What would you need to adapt?

Comment (coolo) [reply .circleci/config.yml]: I'm interested in what needs to be adopted for that.

Comment (DavidKang) [reply .circleci/config.yml]: not much, for sure the checking of the [logo](ttps://github.com/openSUSE/open-build-service/blob/master/src/api/test/functional/webui/spider_test.rb#L108)

Comment (hennevogel) [reply .circleci/config.yml]: I'm all for having the spider test run with what we will use most. Not necessarily in this PR..

**Comment by hennevogel** [comment]
Here we go!!!! :rocket:

**Pull request merged**